securesystemslib/signer.py

```diff
@@ -88,6 +89,64 @@ def to_dict(self) -> Dict:
 
 
 
+class GPGSignature(Signature):
+    """A container class containing information about a gpg signature.
+
+    Besides the signature, it also contains other meta information
+    needed to uniquely identify the key used to generate the signature.
+
+    Attributes:
+        keyid: HEX string used as a unique identifier of the key.
+        signature: HEX string representing the signature.
+        other_headers: HEX representation of additional GPG headers.
+        info: Dict containing information about the GPG key
+            used to generate the signature and the signed data.
+    """
+    def __init__(
+        self,
+        keyid: str,
+        signature: str,
+        other_headers: str,
+        info: dict = None
```

Comment:
I suggest we ignore `info` in this class, it's only needed internally and I don't expect `GPGSignature` to be used internally in `secureystemslib.gpg`.

See 
https://github.com/secure-systems-lab/securesystemslib/blob/a9244ed03cee41c593fb3c2459bf673ca33a8d7d/securesystemslib/gpg/common.py#L689-L692